functions/tree_calculation.py

```diff
@@ -0,0 +1,22 @@
+import colorama
+from colorama import Fore
+colorama.init()
+
+def calc_trees(offset):
+    """1 gram of carbon produces 3.67grams of co2
+       1 tree can absorb 21,772.416 grams of co2 in its life time on an avg
+       :param offset
+       :return %of tree saved """
+    co2_prod= 3.67*offset
+    Tree_saved=(co2_prod/(21772.416))*100

```

Comment:
We will need to change this variable name to all lower case. Anete said we need to make sure our code follows PEP 8. [PEP 8 says variable names need to be in lower case.](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). 

I think a better name would be tree_equivalent or carb_offset_tree_equivalent as we are technically not saving trees, but we are saying that the carbon a user has saved by walking is the equivalent that would be offset by planting this many trees.

Reply:
you are right!! I'm gonna change and update

Reply:
That's the initial draft, I renamed variables and added extra conditions, please go to file changed tab to review the final changed files before requesting or making any changes @robynfsj @Sravya12379

Reply:
oh, I pushed another commit that renamed variable names earlier, could you have a look at that one? here is the link: https://github.com/Walk2Zero/walk2zero/pull/32/commits/e680772f0e3a224a5875241e9032d7e7e224d89a